modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java

```diff
@@ -150,6 +159,34 @@ private static int _updateStartedAppServerStopCounters(
 		return originalCounter;
 	}
 
+	private Configuration _addConfigurationTestModules(final Project project) {
+		Configuration configuration = GradleUtil.addConfiguration(
+			project, TEST_MODULES_CONFIGURATION_NAME);
+
+		configuration.defaultDependencies(
+			new Action<DependencySet>() {
+
+				@Override
+				public void execute(DependencySet dependencySet) {
+					_addDependenciesTestModules(project);
```

Comment:
The code is [here](https://github.com/brianchandotcom/liferay-portal/pull/48816/files#diff-0552859495dfc80071edb3a7cc8865b6R547). I add both the configuration and the `osgi/test` dir as source of that `copy` command, telling Gradle to exclude duplicates.